guide/onboarding/funding-a-wallet.md

```diff
@@ -22,93 +30,205 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
 
 # Funding a wallet
 
-Common problems faced by new users include buying their first bitcoin and transferring it to a new wallet. For this, most will purchase their first bitcoin from an exchange using a national or fiat currency. This generally involves providing some form of documentation (such as ID) and already having a bank account. After purchasing their bitcoin, they can then send it to an external wallet. While this works, there are better alternatives available. Understanding how your users fund their wallets can help you create more tailored and dedicated UI flows. Certain ways users fund their wallets are outlined below.
+The final stage of onboarding users to a Bitcoin wallet is having them fund it with some bitcoin. Below we explore different design directions your application could take when users fund their wallet for the first time.
 
-### Earning bitcoin
+## With Lightning services
 
-Earning bitcoin through exchange of goods and services is a great option. Some individuals and businesses have already started accepting and even paying employees in bitcoin.
+A "Lightning service" is a service wallet providers offer users to avoid having them manage and configure their wallets manually. Some of these services, and the user pain points they solve when funding a wallet, are as followed:
 
-> Fun Fact 🏈: Former Seattle Seahawk Russel Okung puts half of his salary in Bitcoin and is now considered one of the highest paid players in the league.
->
-> <cite>As reported by <a href="https://www.nbcsports.com/northwest/seahawks/former-seattle-seahawk-russell-okung-puts-half-salary-bitcoin-considered-highest">NBC Sports</a></cite>
+* Pay-to-open provides on-demand inbound payment channel liquidity so users can fund their wallet with Lightning payments straight away.
+* Swaps so users can open a payment channel and fund their wallet with an on-chain transfer. 
+* Spend-unconfirmed so users can send and receive as soon as their wallet is funded without having to wait for the on-chain transaction to confirm.
```

Comment:
I think zero confirmation is clearer here? And it's already used term in wallets like Breez.
```suggestion
* Zero-confirmation so users can send and receive as soon as their wallet is funded without having to wait for the on-chain transaction to confirm.
```

Reply:
They call them turbo channels in Breeze, which I'm not a fan. Zero-confirmation doesn't really give much of a description to what that service is offering. Spend-unconfirmed is used by Phoenix and I think is a bit more descriptive to what the service is actually enabling.